projects/timer_challenge/src/main.c

```diff
@@ -0,0 +1,41 @@
+// Firmware 102 Homework
```

Comment:
I think putting your name and labeling it is nice. But for actual projects, we normally don't sign and date them.

I know companies like Google and Qualcomm will keep a log of changes for all their code in the source files for certain projects, but at Midnight Sun we don't do this.
Maybe we could in the future, but for now this isn't part of our coding standards